acq4/devices/FilterWheel/FilterWheel.py

```diff
@@ -0,0 +1,402 @@
+# -*- coding: utf-8 -*-
+from acq4.devices.OptomechDevice import *
```

Comment:
Usually preferred to avoid `import *`, since this makes it difficult to determine where symbols came from and can have unintended side-effects.